sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -501,6 +571,11 @@ public void run() {
                 eventLog.setId(UUIDUtils.randomUUID());
                 eventLog.setName(name);
                 eventLog.setProperties(properties);
+                AnalyticsTransmissionTarget aTransmissionTarget = (transmissionTarget == null) ? mDefaultTransmissionTarget : transmissionTarget;
+                if (aTransmissionTarget != null) {
+                    eventLog.addTransmissionTarget(aTransmissionTarget.mTransmissionTargetToken);
+                    // TODO add multiple targets
```

Comment:
Prefer `/* */` we use `//` only to disable some warnings when an annotation is unavailable.